book/data_sources.qmd

```diff
@@ -1,37 +1,32 @@
+# Data sources
 
-# Data Sources
+This page tabulates and provides links to all data sources used in this project.
+For more details on how each dataset is used and processed, refer to the [Methodology](method.qmd) section.
 
-The list of data sources used in this project. For more details on how is each dataset
-used and processed, refer to the [Methodology](method.qmd).
+## Project-wide
 
-## List of data source
+- Ordnance Survey (OS) | 2011 Census Output Areas | [link](https://geoportal.statistics.gov.uk/datasets/ons::output-areas-dec-2011-boundaries-ew-bgc/about)
 
-Below is a list of data sources organised by the purpose
+## Accessibility indicators
 
-### Project-wide
-
-- OS | 2011 Census Output Areas | [link](https://geoportal.statistics.gov.uk/datasets/ons::output-areas-dec-2011-boundaries-ew-bgc/about)
-
-### Accessibility indicators
-
-- ONS | Population Weighted Centroids (PWC) of Output Areas | [link](https://www.ons.gov.uk/census/2001censusandearlier/dataandproducts/outputgeography/outputareas)
+- Office for National Statistics (ONS) | Population Weighted Centroids (PWC) of Output Areas | [link](https://www.ons.gov.uk/census/2001censusandearlier/dataandproducts/outputgeography/outputareas)
```

Comment:
The PWC link is just an explanation of OAs, it doesn't have any PWCs.

I did find a link to 2011 OA PWCs: https://geoportal.statistics.gov.uk/datasets/ons::output-areas-dec-2011-pwc/about

Is this the correct dataset?

Reply:
The latest update to this file was done by @martinfleis and I guess he'd have the last word, but these data is used for the accessibility indicator, so I guess the link was initially given by me.
My guess is that we wanted to point to an explanation of the different administrative units (as in Output Areas VS Working Place Zones) rather than to the data itself, as the information we wanted to give here was about the existing geometries, and this came from the context where I have applied it (IE in some notebook I explain why I decide to introduce WPZs instead of OAs for accessibility calculations). Also, I see the value of pointing to the explanation rather than the data itself, as the link to the specific data is static so the link could get broken once an updated version comes out; also a user could opt for that a newer/older version of the same data.
So, I see the reason why we could keep the original link, but I can understand also that, if this is a list of links to the data sources, we could point to the actual repo.

Reply:
We want to point to data itself when possible, imo. If we could point to actual PWC here that would be great. @ciupava do you have the link you used?

Reply:
For the data link itself, we should link to the exact version that was used — that would let people reproduce the research exactly. If a user wants to use older or newer data, that would be on them.

I certainly think explanations are really useful! It might be a little bit lost amongst this table of links, though. Perhaps we could place it in the methodology page?

I'd be happy to do that when I get around to that page (after we clear this bunch of comments).

Reply:
Above is correct